client/controllers/PlanningController.tsx

```diff
@@ -63,7 +74,12 @@ export class PlanningController {
         if (!workspaceChanged) {
             return Promise.resolve();
         }
-
+        if (this.store) {
```

Comment:
You could call `this.onMenuChange(false)` instead of duplicating that code here

Reply:
A quick comment here could help as well, as to why we're manually setting the `MENU_OPEN` to false.
That way in the future we know why this code was added

Reply:
Okay.